Corona-Warn-App/src/main/res/values/legal_strings.xml

```diff
@@ -166,4 +166,20 @@
     <string name="rat_profile_onboarding_card_consent6" translatable="false">The QR code contains your personal information. Do not provide your personal QR code to anyone if you do not want the data to be read. We do not recommend publishing your personal QR code or sending it by email.</string>
     <!-- YMSG: RAT Profile Onboarding bullet points -->
     <string name="rat_profile_onboarding_card_consent7" translatable="false">Private individuals as well as companies are not allowed to make you share your rapid test profile with them.</string>
+
+    <!-- Vaccination Consent Privacy Strings  -->
+    <!-- XHED: Title for privacy card -->
+    <string name="vaccination_privacy_card_title_text" translatable="false">"Ihr Einverständnis"</string>
+    <!-- XTXT: Subtitle for privacy card -->
+    <string name="vaccination_privacy_card_subtitle_text" translatable="false">"Durch Antippen von „Einverstanden“ willigen Sie in folgende Schritte ein:"</string>
+    <!-- XTXT: First bulletpoint title for privacy card -->
+    <string name="vaccination_privacy_card_first_bulletpoint_title_text" translatable="false">"Die App legal"</string>
+    <!-- XTXT: First bulletpoint subtitle for privacy card -->
+    <string name="vaccination_privacy_card_first_bulletpoint_subtitle_text" translatable="false">"Wenn Sie es sich später anders überlegen, können Sie den Test in der App entfernen."</string>
+    <!-- XTXT: First bulletpoint title for privacy card -->
+    <string name="vaccination_privacy_card_second_bulletpoint_title_text" translatable="false">"Wenn Sie positiv auf Corona getestet wurden, teilt die App Ihr Testergebnis, um Nutzer, denen Sie begegnet sind, zu warnen. Dies betrifft Nutzer von Corona-Apps der oben genannten Länder und Nutzer, die zeitgleich am selben Event oder Ort wie Sie eingecheckt waren. Im Falle von Schnelltests werden nur Nutzer der Corona-Warn-App gewarnt."</string>
+    <!-- XTXT: First bulletpoint title for privacy card -->
+    <string name="vaccination_privacy_card_third_bulletpoint_title_text" translatable="false">"Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese geteilt. "</string>
+    <!-- XTXT: Subtitle for privacy card -->
+    <string name="vaccination_privacy_card_body" translatable="false">"Sie können Ihr Einverständnis jederzeit zurücknehmen. Die Einstellung hierfür finden Sie unter „Test anzeigen“. Vor dem Teilen werden Sie nochmal auf Ihr Einverständnis hingewiesen."</string>
```

Comment:
what about DE and TR legal texts?

Reply:
DE legal text has been already updated as copy legal text. However, legal texts are not final and should be updated in future PR.